core/src/test/scala/org/apache/spark/util/collection/unsafe/sort/RadixSortSuite.scala

```diff
@@ -40,23 +41,37 @@ class RadixSortSuite extends SparkFunSuite with Logging {
   case class RadixSortType(
     name: String,
     referenceComparator: PrefixComparator,
-    startByteIdx: Int, endByteIdx: Int, descending: Boolean, signed: Boolean)
+    startByteIdx: Int, endByteIdx: Int, descending: Boolean, signed: Boolean, nullFirst: Boolean)
```

Comment:
Can you at least verify that the existing PR passes with SortSuite randomized tests? The prefix comparator code is very subtle and only SortSuite really tests it well.